platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminAlertImpl.java

```diff
@@ -75,6 +76,15 @@ public AdminAlertImpl(PersistentStore persistentStore, EventAdmin eventAdmin)
       }
     } catch (PersistenceException pe) {
       LOGGER.debug("Error retrieving system alert.", pe);
+
+      return Collections.singletonList(
```

Comment:
❗️ There is a difference in the collection that is returned normally versus this one. The collection returned here is immutable whereas normally it ain't. This could trigger issues from a client of this API as they would not think they could modify it. We should probably simply add to `alerts` and return that list normally.

Reply:
What you describe is possible. It would be concerning if callers of the method were relying on the fact that the return object was/was not mutable. It is an assumption I don't think client code is entitled to make.

Reply:
Since it is not documented in anyway, I don't think we can assume that. Better be safe than sorry.